### PR TITLE
Add GTFS+ validation issue details

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [na] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented (no new public methods/classes)
- [na] The description lists all applicable issues this PR seeks to resolve
- [na] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

(EDITED) This PR adds validation details to the GTFS+ validation view. Visual changes are shown in the screenshots attached.

image

Visual changes include:
- GTFS+ files are sorted alphabetically.
- Table-level issues now mention the applicable field name(s).

### CODE REVIEW

File: lib/gtfsplus/components/GtfsPlusVersionSummary.js [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
+  Row,
+  Table
+} from 'react-bootstrap'
 import {browserHistory, Link} from 'react-router'
-import moment from 'moment'
 
-import {getGtfsPlusSpec} from '../../common/util/config'
 import * as gtfsPlusActions from '../actions/gtfsplus'
-
+import {getGtfsPlusSpec} from '../../common/util/config'
```
Comment (landonreed) [inline lib/gtfsplus/components/GtfsPlusVersionSummary.js:18]: Can you put a new line between this import and the type imports? We like to keep those separate.

File: lib/gtfsplus/components/GtfsPlusVersionSummary.js [hunk 2]
```diff
@@ -1,16 +1,31 @@
...
 import type {Props as ContainerProps} from '../containers/ActiveGtfsPlusVersionSummary'
 import type {GtfsPlusReducerState, ManagerUserState} from '../../types/reducers'
 
+type Issue = {
+  description: string,
+  fieldName: string,
+  rowIndex: number,
+  tableId: string
+}
+
```
Comment (landonreed) [inline lib/gtfsplus/components/GtfsPlusVersionSummary.js:28]: This is already defined in `types/index.js` as `GtfsPlusValidationIssue`.

File: lib/gtfsplus/components/GtfsPlusVersionSummary.js [hunk 3]
```diff
@@ -190,43 +205,45 @@ export default class GtfsPlusVersionSummary extends Component<Props, State> {
...
-                              <br />
-                              {tableLevelIssues.length} critical table issue(s):
-                              <ul>
-                                {tableLevelIssues.map((issue, i) =>
-                                  <li key={i}>{issue.description}</li>)}
-                              </ul>
-                            </small>
-                            : null
+                  {getGtfsPlusSpec()
+                    .sort((table1, table2) => table1.name.localeCompare(table2.name))
```
Comment (landonreed) [inline lib/gtfsplus/components/GtfsPlusVersionSummary.js:212]: Shouldn't we just sort the gtfs.yml file keys instead?

Comment (binh-dam-ibigroup) [reply lib/gtfsplus/components/GtfsPlusVersionSummary.js:212]: Sort that file too I'd say.

**Pull request merged**